beagle_etl/jobs/lims_etl_jobs.py

```diff
@@ -331,39 +331,31 @@ def fetch_sample_metadata(sample_id, igocomplete, request_id, request_metadata):
                 logger.error("Failed to fetch SampleManifest for sampleId:%s. Fastqs empty" % sample_id)
                 missing_fastq = True
                 failed_runs.append(run['runId'])
-            elif len(fastqs) != 2:
+            elif len(fastqs) % 2 != 0:
                 logger.error(
                     "Failed to fetch SampleManifest for sampleId:%s. %s fastq file(s) provided" % (
                     sample_id, str(len(fastqs))))
                 invalid_number_of_fastq = True
                 failed_runs.append(run['runId'])
             else:
-                file_search = FileRepository.filter(path=fastqs[0]).first()
-                if not file_search:
-                    create_file(fastqs[0], request_id, settings.IMPORT_FILE_GROUP, 'fastq', igocomplete, data, library, run,
-                                request_metadata, R1_or_R2(fastqs[0]))
-                else:
-                    logger.error(
-                        "File %s already created with id:%s" % (file_search.file.path, str(file_search.file.id)))
-                    conflict = True
-                    conflict_files.append((file_search.file.path, str(file_search.file.id)))
-                file_search = FileRepository.filter(path=fastqs[1]).first()
-                if not file_search:
-                    create_file(fastqs[1], request_id, settings.IMPORT_FILE_GROUP, 'fastq', igocomplete, data, library, run,
-                                request_metadata, R1_or_R2(fastqs[1]))
-                else:
-                    logger.error(
-                        "File %s already created with id:%s" % (file_search.file.path, str(file_search.file.id)))
-                    conflict = True
-                    conflict_files.append((file_search.file.path, str(file_search.file.id)))
-    if conflict:
```

Comment:
Removed this piece so that the runs don't get reported as a FAILED if we see the file again during the same import but we may need to think about how to handle this better